crypto/rand/randfile.c

```diff
@@ -7,6 +7,13 @@
  * https://www.openssl.org/source/license.html
  */
 
+#if defined (__TANDEM) && defined (_SPT_MODEL_)
+/* These definitions have to come first in SPT due to scoping of the
+ * declarations in c99 associated with SPT use of stat. */
```

Comment:
Our code style says that block comments shall look like this:

``` C
/*
 * These definitions have to come first in SPT due to scoping of the
 * declarations in c99 associated with SPT use of stat.
 */
```

Reply:
fixed and pushed